backend/packages/Upgrade/src/api/repositories/ExperimentAuditLogRepository.ts

```diff
@@ -35,9 +35,17 @@ export class ExperimentAuditLogRepository extends Repository<ExperimentAuditLog>
       });
   }
 
-  public async saveRawJson(type: EXPERIMENT_LOG_TYPE, data: any, user: User): Promise<ExperimentAuditLog> {
-    const result = await this.createQueryBuilder()
+  public async saveRawJson(
+    type: EXPERIMENT_LOG_TYPE,
+    data: any,
+    user: User,
+    entityManger?: EntityManager
+  ): Promise<ExperimentAuditLog> {
+    const that = entityManger ? entityManger : this; 
```

Comment:
I think we can update this later by OR operator also.
`const that = entityManager || this`

Reply:
Code Updated